apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -0,0 +1,316 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-sso-saml',
+  title: 'Single Sign-On with SAML 2.0',
+  description: 'Use Single Sign-On (SSO) authentication with SAML 2.0',
+  video: 'https://www.youtube.com/v/em1cpOAXknM',
+}
+
+Supabase Auth supports enterprise level Single Sign-On (SSO) using the SAML 2.0 protocol. In this document you will learn how you can manage connections to compatible identity providers.
```

Comment:
```suggestion
Supabase Auth supports enterprise-level Single Sign-On (SSO) for any Provides compatible with the using the SAML 2.0 protocol.
```

Reply:
not sure exactly about this wording, but generally I like to avoid saying "what you will learn" - just get straight into it